src/control/cmd/daos_server/main.go

```diff
@@ -149,6 +138,9 @@ func main() {
 		exitWithError(log, err)
 	}
 
+	// Set the package-level logger for netdetect.
+	netdetect.SetLogger(log)
+
```

Comment:
Do we need this here? netdetect isn't used for any network stuff anymore, and I have a PR up to remove it completely. It just needs to get through CI's various foibles.

Reply:
Hmm, nope. Happy to rebase once yours lands.